nims-notes-things/src/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "styled-components";
+
+const Header = (props) => {
+  const { type } = props;
```

Comment:
what exactly is the type doing here?

Reply:
> what exactly is the type doing here?

I thought the type would be used to set the header as h1, h2 etc but from our conversation i can just spread the props